Manual symlink of libzmq.so.5 required for pyzmq

chapmanb opened the pull request:
Fixes #16

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

SylvainCorlay commented:
This looks good to me. I will merge as soon as all the boxes are checked.

jakirkham commented on recipe/build.sh:
```diff
@@ -14,6 +14,10 @@ cd build
 
+# Add missing symlink for libzmq.so.5 required for pyzmq
+# https://github.com/conda-forge/zeromq-feedstock/issues/16
+ln -f -r -s $PREFIX/lib/libzmq.so $PREFIX/lib/libzmq.so.5
```
Could you please replace `.so` with `${SHLIB_EXT}`? On Mac these are `dylib`s. `conda-build` sets `$SHLIB_EXT` to be the right extension depending on the platform.

jakirkham replied:
Also what is the `-r` option for?

jakirkham commented:
Just to make sure we don't lose this file in the future, would you mind adding a file test too?

chapmanb commented:
Thanks for all the suggestions. I updated to use the generic file extension, avoiding using `-r` for relative path resolution and added a file test. I won't be available for updates to this for a while so if there are other issues feel free to take this as a base and merge in a fixed version. Much appreciated.

jakirkham commented:
Thanks @chapmanb.

jakirkham commented on recipe/build.sh:
```diff
@@ -14,6 +14,10 @@ cd build
 
+# Add missing symlink for libzmq.so.5 required for pyzmq
+# https://github.com/conda-forge/zeromq-feedstock/issues/16
+ln -f -s libzmq.${SHLIB_EXT} $PREFIX/lib/libzmq.${SHLIB_EXT}.5
```
`$SHLIB_EXT` contains a `.` already. Will push a change to your PR.

jakirkham replied:
Fixed.

chapmanb commented:
Thanks so much for all the additional fixes and test. I appreciate the help getting this ready so quickly.
